app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Orgs
+  class LtiConfigurationsController < Orgs::Controller
+    before_action :ensure_lti_launch_flipper_is_enabled
+    before_action :ensure_current_lti_configuration, except: %i[new create]
+
+    def create
+      # TODO: Create a new lti configuration for the current organization here
+      # TODO: redirect_to lti_configuration_path(current_organization) after
+    end
+
```

Comment:
Nit: In this project we usually implement empty methods for actions which have routes. This slightly improves readability.
```suggestion
    def show; end

    def new; end

```